src/main/java/duke/commands/CreateTodo.java

```diff
@@ -0,0 +1,36 @@
+package duke.commands;
+
+import java.io.IOException;
+
+import duke.ui.Ui;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+import duke.tasks.TaskList;
+import duke.storage.Storage;
+import duke.exceptions.DukeException;
+
+/**
+ * Creates a Todo Task and adds it to the TaskList.
+ */
+class CreateTodo implements Command {
+    public void execute(String arg, TaskList tasks, Ui ui, Storage storage) throws DukeException {
+        if (arg.length() == 0) {
+            throw new DukeException("Usage: todo [task name]");
+        }
+
+        // Create new Task
```

Comment:
Isn't this obvious looking at the code? you can eliminate comments that state the obvious.